docs/users/extend/additional-services.md

```diff
@@ -18,6 +18,14 @@ This recipe adds an Apache Solr 5.4 container to a project. It will setup a solr
 - To access the Solr container from the web container use `http://solr:8983/solr/`
 - The Solr core will be "dev"
 
+### Drupal's Search API configuration
```

Comment:
This seems to add very Drupal-centric stuff to the docs, which aren't a Drupal tutorial at all. Would this be better as a full Drupal SOLR tutorial on Stack Overflow or a blog? I note also @mglaman 's tutorial on multiple cores for Drupal, https://glamanate.com/blog/using-multiple-solr-cores-ddev?utm_source=Drupal%20Planet&utm_medium=feed&utm_campaign=drupal

Reply:
Sure, that can be changed and moved to SO. I can create that.